src/components/Asset/Edit/index.tsx

```diff
@@ -37,13 +37,17 @@ export default function Edit({ uri }: { uri: string }): ReactElement {
     }
   ].filter((tab) => tab !== undefined)
 
-  return asset && asset?.accessDetails && accountId === owner ? (
+  return asset &&
+    asset?.accessDetails &&
+    accountId?.toLowerCase() === owner?.toLowerCase() ? (
```

Comment:
I guess is fine, but why are we getting this ID formatted differently?

Reply:
I'm not really sure to be honest